doc/installation.xml

```diff
@@ -225,6 +225,45 @@ home-manager.useUserPackages = true;
     become the default value in the future.
    </para>
   </note>
+
+  <note>
+   <para>
+    By default, Home Manager modules use a private <literal>pkgs</literal> instance
+    that is configured via the <option>nixpkgs</option> options.
+    To disable these options and to use the global <literal>pkgs</literal> from
+    the NixOS modules, set
+   </para>
+<programlisting language="nix">
+home-manager.useGlobalPkgs = true;
+</programlisting>
+  <para>
+   This saves an extra nixpkgs evaluation, adds consistency, and removes the
+   dependency on <literal>NIX_PATH</literal>, which is otherwise used for
+   importing the pkgs.
+   This option may become the default value in the future.
```

Comment:
```suggestion
   This saves an extra Nixpkgs evaluation, adds consistency, and removes the
   dependency on <envvar>NIX_PATH</envvar>, which is otherwise used for
   importing Nixpkgs.
```

Reply:
No, `envar` from your deleted comment is actually correct :smiley_cat:

Reply:
I always forget which way it is 🙂